src/logic/calculate.js

```diff
@@ -0,0 +1,47 @@
+import operate from './operate';
```

Comment:
you should import this file in `App` component.
![image](https://user-images.githubusercontent.com/31897434/79172766-83317880-7ded-11ea-9f29-dc9d4b15b92f.png)

Reply:
My bad  I totally forgot this, sorry I will fix it and thanks again !